README.md

```diff
@@ -97,6 +97,20 @@ We use Code generation to generate our types for our GraphQL queries and mutatio
 - Each query and mutation should have a unique name.
 - Since query analysis for type generation occurs statically we cant place dynamic variables with in query strings we instead have to hard code the variable in the query or pass it in as query variable.
 
+## How to get data for your feature
+If you need more data to be able to test out your feature locally the easiest way to do it is to populate the local db using real data from the staging or production environments.
+- First step is to locate where you want to fetch your data from if it exists in either the staging or production environments.
+- After identifying where to get your data from you should try to ssh into one of the db servers used for staging or prod. The urls for these dbs can be located in the `fabfile.py` located in the evergreen directory.
+- You should make sure you are connected to a secondary db before proceeding.
+- Open the mongo shell and identify what query returns the data you need.
```

Comment:
nit: I thought these 2 steps were kind of confusing since I may already know the collection and query for the data I need. If we decide to keep these lines, I think we should mention something about identifying the collection along with the query.